samples/todolist.md

```diff
@@ -0,0 +1,160 @@
+# Pelorus Samples & Use Cases: Todolist
+
+## Deploying a todolist Sample alongside the Webhook/Generic CommitTime Exporter
+
+* The Pelorus exporter requires the Pelorus git source and branch
+  * Fork [this repo](https://github.com/konveyor/pelorus) to https://github.com/your_org/pelorus
+* We'll need a sample application to measure the commits and deployments
+  *  Fork [this repo](https://github.com/konveyor/mig-demo-apps) to https://github.com/your_org/mig-demo-apps
+  * Note: We'll be using the todolist-mongo-go application
+* The definitive Pelorus install guide can be found [here](https://pelorus.readthedocs.io/en/stable/Install/)
+
+## Pelorus install steps
+1. Create the pelorus namespace
+```
+oc create namespace pelorus
+```
+2. Install the required granfa and prometheus operators
+```
+helm install operators charts/operators --namespace pelorus
+```
+3. Create a GitHub Personal Access Token and store it as a secret in Openshift:
+```
+oc create secret generic github-secret --from-literal=GIT_USER=<username> --from-literal=GIT_TOKEN=<personal access token> --namespace pelorus
```

Comment:
The GitHub exporter actually doesn't need a token if the repo is public!

Reply:
aye.. same as above